documentation/modules/exploit/linux/http/apache_hugegraph_gremlin_rce.md

```diff
@@ -0,0 +1,46 @@
+## Vulnerable Application
+This module exploits CVE-2024-27348 which is a Remote Code Execution (RCE) vulnerability that exists in
+Apache HugeGraph Server in versions before 1.3.0. An attacker can bypass the sandbox restrictions and achieve
+RCE through Gremlin, resulting in complete control over the server
+
+### Setup
+To install a vulnerable instance via docker run the following command:
+```
+docker run -itd --name=graph -p 8080:8080 hugegraph/hugegraph:1.0.0
+```
+
+## Verification Steps
+
+1. Start msfconsole
+1. Do: `use exploit/multi/http/apache_hugegraph_gremlin_rce`
+1. Set the `RHOST` and `LHOST` options
+1. Run the module
+1. Receive a Meterpreter session as the `root` user.
+
+## Scenarios
+### Apache HugeGraph 1.0.0 docker instance
+```
+
+msf6 exploit(multi/http/apache_hugegraph_gremlin_rce) > set rhost 127.0.0.1
```

Comment:
If the module path does need updated, just adding a reminder to update this section as well.